client/environment_import_test.go

```diff
@@ -0,0 +1,134 @@
+package client_test
+
+import (
+	. "github.com/env0/terraform-provider-env0/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/mock/gomock"
+)
+
+var _ = Describe("Environment Import Client", func() {
+	mockEnvironmentImport := EnvironmentImport{
+		Id:         "id",
+		Name:       "name",
+		IacType:    "tofu",
+		IacVersion: "1.0",
+		Workspace:  "workspace",
+		Variables: []Variable{{
+			Name:        "name",
+			Value:       "value",
+			IsSensitive: false,
+			Type:        "string",
+		},
+		},
+	}
+
+	Describe("EnvironmentImportGet", func() {
+		var result *EnvironmentImport
+
+		BeforeEach(func() {
+			mockHttpClient.EXPECT().
+				Get("/staging-environments/"+mockEnvironmentImport.Id, nil, gomock.Any()).
+				Do(func(path string, request interface{}, response *EnvironmentImport) {
+					*response = mockEnvironmentImport
+				})
+			result, _ = apiClient.EnvironmentImportGet(mockEnvironmentImport.Id)
+		})
+
+		It("Should get organization id", func() {
```

Comment:
remove this...

Reply:
[1f82cc2](https://github.com/env0/terraform-provider-env0/pull/855/commits/1f82cc2273d00e9005519eb32d6689e1d117b2bd)